lib/internal/vm/module.js

```diff
@@ -194,7 +195,25 @@ class Module {
     wrap.instantiate();
   }
 
-  async evaluate(options) {
+  async evaluate(options = {}) {
+    if (typeof options !== 'object' || options === null) {
+      throw new ERR_INVALID_ARG_TYPE('options', 'Object', options);
+    }
+
+    let timeout = options.timeout;
+    if (timeout === undefined) {
+      timeout = -1;
+    } else if (!Number.isInteger(timeout) || timeout <= 0) {
```

Comment:
Should timeout also run through validateInteger?

Reply:
The check is a bit different. `timeout` may not be negative, while validateInteger allows negative values.